### ISSUE TITLE
backTitleVisible cannot be forced to false

### ISSUE THREAD

**Issue by Jakst** [react-navigation/react-navigation-stack#2]
HeaderBackButton.js
```javascript
if (titleDefaultsToDisabled && !backTitleVisible) {
   return null;
}
```

This piece of code does not respect an explicitly set `backTitleVisible: false`.

So if I set `headerBackTitleVisible: false` in my StackNavigator with react-navigation, the back title is not hidden on iOS.

**Comment by brentvatne** [comment]
can you submit a PR to fix this?

**Comment by brentvatne** [comment]
this is resolved in the 1.0 alpha version of react-navigation-stack. you can use that now if you like or wait until react-navigation 3 to be released in mid october
